crates/block-explorers/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![doc = include_str!("../README.md")]
+#![doc = include_str!("../../../README.md")]
```

Comment:
you need to symlink the readme into the crate folder or you can't release it

Reply:
this readme is for block-explorers, I'll just move this readme to the crate and add a high-level readme